src/main/java/org/prebid/server/bidder/interactiveoffers/InteractiveOffersBidder.java

```diff
@@ -38,9 +39,22 @@ public InteractiveOffersBidder(String endpointUrl, JacksonMapper mapper) {
 
     @Override
     public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+
+        ExtImpInteractiveoffers extImpInteractiveoffers = null;
```

Comment:
looks like this redundant, you dont use it

Reply:
changed